es/readers-guide.md

```diff
@@ -18,12 +18,12 @@ Esta sección explica por qué es importante seguir los requisitos para personas
 
 Esta sección gestiona expectativas a través de la explicación de requisitos que el criterio no cumplirá.
 
-Esto ayuda:
+Esto ayuda a:
 
-* con una correcta aplicación del Estándar
-* a asegurarse de que no aparecen questiones improvistas
+* Aplicar de manera correcta el Estándar.
+* Asegurarse de que surjan imprevistos.
 
-## Cómo testear
+## Cómo probar
```

Comment:
I would like to keep the word "Testear". It is in the RAE dictionary (https://dle.rae.es/testear?m=form) and also refers directly to the word 'test', as in perform software checks. Perhaps we can use "hacer pruebas o tests"